pkg/apis/apps/v1alpha1/apicast_types.go

```diff
@@ -71,6 +71,10 @@ type APIcastStatus struct {
 	// +patchMergeKey=type
 	// +patchStrategy=merge
 	Conditions []APIcastCondition `json:"conditions,omitempty" patchStrategy:"merge" patchMergeKey:"type"`
+
+	// The image being used in the APIcast deployment
+	// +optional
+	Image string `json:"image,omitempty"`
```

Comment:
I've set this field as optional and omitempty.
The reason for that is that I've seen other Kubernetes objects that also havfe some status fields that seem they should always be defined but they also are optional,omitempty. For example, The replicas field in a Deployment is optional,omitempty

Reply:
:+1: